src/utils/object.js

```diff
@@ -60,13 +63,13 @@ function deepCollection (collectionId, obj) {
 }
 
 /**
- * isObject function opinionated against Arrays
+ * IsObject function opinionated against Arrays
```

Comment:
It's a little bit awkward the function is named `isObject` but it is referenced as `IsObject` in the comment. 🤔

Reply:
Hm, yeah. I'll see which tool changed that

Reply:
Hm, I'm not sure what the best approach is because the change is caused by this rule: https://eslint.org/docs/rules/capitalized-comments

I think I'm gonna disable this rule for block comments but leave it enabled for line comments. Does this sound OK?

Reply:
I see. Hmm... 🤔 
> I think I'm gonna disable this rule for block comments but leave it enabled for line comments. Does this sound OK?

Maybe you should leave it as-is. It's a bit akward since we are retrofitting some of these linting changes.

Further down the files I see this:
```diff
-//var config = require('./config');
+//Var config = require('./config');
```
If anything, perhaps we should create new GH issues to edit those comments instead of disabling the linting rule? @calebeby What do you think?

Reply:
https://github.com/cloudfour/drizzle-builder/issues/107